src/server/mod.rs

```diff
@@ -137,7 +137,7 @@ impl Server {
 
             vec![build_app(state), build_dockerflow(dfstate)]
         })
-        .bind(format!("127.0.0.1:{}", settings.port))
+        .bind(format!("0.0.0.0:{}", settings.port))
```

Comment:
is the new settings.host actually used anywhere (looks like it should be here)?

Reply:
oh for... forgot to commit that change.